modules/ai/assets/js/editor/components/prompt-history/helpers/history-item-helpers.js

```diff
@@ -0,0 +1,21 @@
+import TextIcon from '../../../icons/text-icon';
+import { AIIcon } from '@elementor/icons';
+import CodeIcon from '../../../icons/code-icon';
+
+const FALLBACK_ICON = <AIIcon data-testid="e-ph-fi" />;
```

Comment:
My intention here and below was to reduce document size which makes sense if we repeat certain elements multiple times.

E. g.:
`e-ph-fi` -- 7 bytes. If we use it 10 times it will be 70 bytes.
`e-prompt-history-fallback-icon` -- 30 bytes; 10 entries is 300 bytes which is almost a half of kilobyte for a thing that brings no value for the end user and is needed only for the tests.

And that is only for one selector, I used plenty of them during development.

Wdyt?

Reply:
Of course, it should be as short as possible, but it's hard to understand it like this (it's like calling it: "xyz" if you want to count bytes :) it's even "better" ), so I don't think that bytes should be the consideration.
+
See the other comment that I wrote in the E2E part.. I'm not sure that it should even be part of the E2E, please check again.